Pinta.Gui.Widgets/Utilities/GradientHelper.cs

```diff
@@ -0,0 +1,122 @@
+using System.Collections.Generic;
+using Pinta.Core;
+
+namespace Pinta.Gui.Widgets;
+
+public enum PredefinedGradients
+{
+	[Caption ("Black and White")]
+	BlackAndWhite,
+
+	[Caption ("Bonfire")]
+	Bonfire,
+
+	[Caption ("Cotton Candy")]
+	CottonCandy,
+
+	[Caption ("Electric")]
+	Electric,
+
+	[Caption ("La Bella Italia")]
+	LaBellaItalia,
+
+	[Caption ("Lime Lemon")]
+	LimeLemon,
+
+	[Caption ("Piña Colada")]
+	PinaColada,
+
+	[Caption ("Sakura Sigh")]
```

Comment:
Was there a standard list of gradients this was sourced from? I'm just wondering in terms of how to describe what these strings mean for translators

Reply:
No, I made them up. The gradients are not based on anything in particular, either; I just created several different ones so that there is some variety in the gradients that the user can choose.

Reply:
OK 👍 
To make this easier for translators, you could add comments like `// Translators: This is the name of a gradient preset which blends from yellow to green` or whatever makes sense to explain it. Comments like that are picked up when the source code is parsed for translatable strings, and made visible on Weblate for translators

I'm not entirely sure how easy it'll be to translate a few of the labels, e.g. `La Bella Italia` and `Sakura Sigh`?

Reply:
No problem, I just changed a couple of names to make them easier to translate, and I also left a few comments for translators (but could you tell me if I did it right?)

Reply:
yep, I think that looks correct 👍